certified-connectors/QPP NG Integration Hub/apiDefinition.swagger.json

```diff
@@ -1753,20 +1827,21 @@
         }
       }
     },
-	"/api/v1/attributeDomainService/getAllDomains": {
+    "/api/v1/attributeDomainService/getAllDomains": {
       "get": {
         "summary": "Get All Domains",
-        "description": "This request is used to get the metadata of all the domains that exist in the Enviroment.Internal Action to facilitate for other action params.",
+        "description": "This request is used to get the metadata of all the domains that exist in the Enviroment.Internal Action to facilitate for other action params.For more information Check the github documentation.",
```

Comment:
Please correct the spelling "Enviroment"

Reply:
Done